Classes/Public/Constants/HTMLConstants.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+public enum HTMLTags: String {
+    case Bold = "b"
+    case Delete = "del"
+    case Emphasized = "em"
+    case Italic = "i"
+    case Link = "a"
+    case StrikeThrough1 = "s"
+    case StrikeThrough2 = "strike"
```

Comment:
This works but would it be better to just use the name of the tag for the name of the enum case?  `HTMLTags.b` (or maybe `HTMLTags.B`?) is just as clear (if not more so) as `HTMLTags.Bold` and it would avoid having to do things like `StrikeThrough1` and `StrikeThrough2`. 
Thoughts?